src/site/layouts/tests/vet_center/fixtures/vet_center_escanaba_data.json

```diff
@@ -336,15 +336,15 @@
         "fieldServiceNameAndDescripti": {
           "entity": {
             "name": "Telehealth",
-            "fieldVetCenterTypeOfCare": "counselling",
+            "fieldVetCenterTypeOfCare": "other",
             "fieldServiceTypeOfCare": "Other services",
```

Comment:
What is the difference bewteen `fieldVetCenterTypeOfCare` and `fieldServiceTypeOfCare`?

Reply:
i have no idea, I am just using what is required according to the mural